modules/popup-stack/README.md

```diff
@@ -205,3 +207,27 @@ interact with it.
 ```tsx
 createAdapter({}: Partial<typeof PopupStack>): void
 ```
+
+## Behaviors
+
+The PopupStack does not directly own any behaviors. Behaviors are left up to the implementation, but
+behavior suggestions are listed here as a specification for implementations. Some behaviors are
+aided by a specification of `data-*` attributes to help those behaviors apply globally across
+frameworks.
+
+### Close on outside click
+
+Close only the topmost popup with a click outside the popup when this behavior is applied. To
+determine if this behavior is applied to the global stack, a
+`data-behavior-click-outside-close="topmost"` should be applied. This data attribute should be used
+by all instances of this behavior to determine the "topmost" click-outside behavior to close only
+the topmost popup with this behavior.
+
+### Always close on outside click
+
+Close all popups with this behavior regardless of the position in the stack. A
+`data-behavior-click-outside-close="always"` should be applied to the stack element.
```

Comment:
The name of the attribute correctly describes what happens when a click outside occurs. It will always close. The "Close all popups" is describing the net effect.

Reply:
Perhaps:

> Always close the popup when the user clicks outside regardless of the position in the stack. A
> `data-behavior-click-outside-close="always"` attribute should be applied to the stack element.